server/cli/src/report_utils.rs

```diff
@@ -8,11 +8,10 @@ use std::{
     path::PathBuf,
     process::{Command, Stdio},
 };
-
 use thiserror::Error as ThisError;
 
 #[derive(ThisError, Debug)]
-pub enum Error {
+pub enum ReportError {
```

Comment:
Have made this error name more explicit because we are now sharing similar error struct with plugin in cli